core/src/main/scala/org/http4s/ContentCoding.scala

```diff
@@ -18,54 +18,117 @@
  */
 package org.http4s
 
-import org.http4s.syntax.string._
+import cats.{Order, Show}
+import cats.syntax.eq._
+import cats.instances.tuple._
+import cats.instances.string._
+import org.http4s.QValue.QValueParser
+import org.http4s.internal.parboiled2.{Parser => PbParser, _}
+import org.http4s.parser.Http4sParser
 import org.http4s.util._
 
-final case class ContentCoding(coding: CaseInsensitiveString, qValue: QValue = QValue.One)
+import scala.util.hashing.MurmurHash3
```

Comment:
Totally pedantic, but our imports are all generally in one block now.  The ordering is correct.